README.md

```diff
@@ -72,10 +72,11 @@ spec:
        - 100:1::1/64
     rules:
     - order: 10
-      protocol: tcp
-      protoRule:
-        ports: "100-200"
-      action: allow
+      protocolConfig:
+        protocol: TCP
```

Comment:
isn't that redundant?
~~~
protocol: TCP  <--
tcp:                  <--
  ports:
~~~

Reply:
this is my understanding of how the union can be used based on this doc 
https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#discriminated-unions

Reply:
nope, absolutely my bad. I wasn't aware of that guide :-)